lib/config/presets/internal/workarounds.ts

```diff
@@ -32,6 +33,16 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  doNotExceedRubygemsRateLimit: {
+    description:
+      'Limit concurrent requests to not exceed the rubygems rate limit.',
+    hostRules: [
+      {
+        matchHost: 'rubygems.org',
+        maxRequestsPerSecond: 15,
+      },
+    ],
+  },
```

Comment:
I think we should have this type of thing as default/embedded when we know a certain registry doesn't want to be overloaded